autotest/README.markdown

```diff
@@ -1,17 +1,17 @@
 # Continuous Testing
 
-This directory contains configurations for continuous testing, using
-either autotest (part of the ZenTest gem) or watchr (in the watchr
-gem).  The purpose of these tools is to automatically run the
-appropriate test when a file is changed or, if appropriate, all
-tests.  In general, they do a straightforward mapping from a given
-code file to its unit and/or integration test.
-
-It is highly recommended that you have one of these running at all
-times during development, as they provide immediate and continuous
-feedback as to your development process.  There are some general
-usability downsides as you have to track the running process, but
-those downsides are easily worth it.
+This directory contains configurations for continuous testing,
+using autotest (part of the ZenTest gem).  The purpose of this
+tool is to automatically run the appropriate test when a file is
+changed or, if appropriate, all tests.  In general, it does a
+straightforward mapping from a given code file to its unit and/or
+integration test.
+
+It is highly recommended that you have some sort of automatic test
+runner running at all times during development, as it will provide
```

Comment:
what's an `automatic test runner`? Do you have any that you'd recommend? Is there somewhere we can find more information about this?

This totally isn't a blocker, but if we're updating this, it might be nice to expand on some of this to make it easier on our users

Reply:
It seemed like the previous paragraph provided the context for what an "automatic test runner" is. `autotest/README.markdown` itself is supposed to be providing a recommendation for which one to use.

You have anything specific you'd like to see to make it better/more clear?

Reply:
This is probably not terribly relevant anymore since it looks like the `autotest` directory and all of its content is going away.

Reply:
👍